docs/src/02-metadata-merging.jmd

```diff
@@ -72,6 +74,9 @@ since we can have multiple samples per timepoint.
 `sampleID`s should be unique though.
 
 ```julia
+#**Error message here**
+# warning: only found 38 / 39 columns on data row: 1194. Filling remaining co
+lumns with `missing`
```

Comment:
Hmm... I have no idea how this can happen. I think it's fine for now, but worth coming back to at some point.